api/app/signals/apps/signals/tasks.py

```diff
@@ -86,3 +87,22 @@ def anonymize_reporter(reporter_id):
                 'text': text,
                 'created_by': None  # This wil show as "SIA systeem"
             }, signal=reporter.signal)
+
+
+@app.task
+def update_status_children_based_on_parent(signal_id):
+    if not settings.FEATURE_FLAGS.get('TASK_UPDATE_CHILDREN_BASED_ON_PARENT', True):
+        # Feature disabled
+        log.warning("Feature 'TASK_UPDATE_CHILDREN_BASED_ON_PARENT' disabled!")
+        return
+
+    text = 'Hoofdmelding is afgehandeld hierdoor is deze deelmelding geannuleerd'
+    signal = Signal.objects.get(pk=signal_id)
+    if signal.is_parent() and signal.status.state in [AFGEHANDELD, ]:
```

Comment:
That is not a problem at all with this setup. Because a "split" melding is also a parent of children.
Only thing could be the message that we add to the status, TODO is to ask the business what message they want to see when this happens. Or perhaps no message at all.

Reply:
Done